liquibase-core/src/main/java/liquibase/change/core/CreateProcedureChange.java

```diff
@@ -132,7 +121,10 @@ public void setProcedureBody(String procedureText) {
         this.procedureText = procedureText;
     }
 
-    @DatabaseChangeProperty(isChangeProperty = false)
+    @DatabaseChangeProperty(
+        description = "The SQL creating the procedure. You need to define either this attribute or 'path'. " +
```

Comment:
It is true that the XML changelog does not have an attribute named `procedureText`. The `procedureText` is implicitly defined as "whatever appears in between `<createProcedure procedureName="name">` and `</createProcedure>`".  For example: 

```
   <changeSet author="lbuser" id="3-proc" labels="community" runOnChange="true" contextFilter="ctx">
        <createProcedure procedureName="simple_proc3">
			ANYTHING HERE IS THE procedureText
		</createProcedure>
    </changeSet>
```

I'm guessing that this implicit definition is what is meant by "nested" but that phrasing confused me. Perhaps, "inline" is better than "nested?"

Reply:
INFO: [DAT-12813 PRO :: "procedureText" attribute doesn't work in JSON and YML changelogs](https://datical.atlassian.net/browse/DAT-12813)

Reply:
I think we should have a similar PD ticket for this because only `procedureText` should be a valid attribute for `CreateProcedureChange`. Both get/set of `procedureBody` are deprecated fields (we should probably remove them from the code as well), so YAML documentation should be updated to reflect this, as well as the code if somehow it is still accepting it. Thank you for pointing this out, @TymofiiKritsak!